mlrun/feature_store/feature_set.py

```diff
@@ -453,13 +454,40 @@ def graph(self):
         """feature set transformation graph/DAG"""
         return self.spec.graph
 
+    def _add_agregation_to_existing(self, new_aggregation):
+        name = new_aggregation["name"]
+        if name in self._aggregations:
+            current_aggr = self._aggregations[name]
+            if current_aggr["windows"] != new_aggregation["windows"]:
+                raise mlrun.errors.MLRunInvalidArgumentError(
+                    f"Aggregation with name {name} already exists but with window {current_aggr['windows']}. "
+                    f"Please provide name for the aggregation"
+                )
+            if current_aggr["period"] != new_aggregation["period"]:
+                raise mlrun.errors.MLRunInvalidArgumentError(
+                    f"Aggregation with name {name} already exists but with period {current_aggr['period']}. "
+                    f"Please provide name for the aggregation"
+                )
+            if current_aggr["column"] != new_aggregation["column"]:
+                raise mlrun.errors.MLRunInvalidArgumentError(
+                    f"Aggregation with name {name} already exists but for different column {current_aggr['column']}. "
+                    f"Please provide name for the aggregation"
+                )
+            current_aggr["operations"] = list(
+                set(current_aggr["operations"] + new_aggregation["operations"])
+            )
+
+            return False
+        self._aggregations[name] = new_aggregation
+        return True
+
     def add_aggregation(
         self,
-        name,
         column,
         operations,
         windows,
         period=None,
+        name=None,
```

Comment:
Why not put it either in the end of params or as the 1st optional parameter? Any reason why this is placed here?